sonar-project.properties

```diff
@@ -1,6 +1,7 @@
 sonar.organization=green-code-initiative
```

Comment:
file really used ? I don't know, because already present when I arrived ... thus, to check.

Reply:
Yes because the project struture is very specific (a folder with the ESLint plugin and another with the Sonar plugin), so this file explains where source files and test files are